dp/cloud/python/magma/configuration_controller/response_processor/response_db_processor.py

```diff
@@ -121,7 +126,11 @@ def _process_responses(
                 f"[{self.response_type}] Adding Response: {db_response} for Request {db_request}",
             )
             session.add(db_response)
-            self._log_response(session, db_response)
+            try:
```

Comment:
IMO retain the `_log_response` method, put this `try/except` block inside that method. So this block of code will look the same, just the private method implementation will change - aka encapsulate the implementation detail.

Reply:
done